retailer-oas/retailer.yaml

```diff
@@ -376,58 +519,170 @@ paths:
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    description: Error message
+                  errorCode:
+                    type: string
+                    description: Error code
+                    enum:
+                      - "TIMESLOT_NOT_AVAILABLE"
+                      - "ORDER_TOO_HEAVY"
+                      - "INVALID_SHIPPING_ADDRESS"
+                      - "INVALID_CUSTOMER_DATA"
+                      - "ORDER_TOO_HEAVY"
+                      - "MAX_ORDERS_EXCEEDED"
+                      - "INVALID_PRODUCT_QUANTITY"
+                      - "ORDER_TOTAL_MISMATCH"
+                      - "DUPLICATE_ORDER"
+                  errors:
+                    type: array
+                    items:
+                      type: string
+                      description: List of error details
 
 components:
-  schemas:      
+  schemas:
+
     AccessToken:
       type: object
       properties:
-        token: 
+        token:
           type: string
-        tokenType: 
+        tokenType:
           type: string
-        scope: 
+        scope:
           type: string
-        expiresIn: 
+        expiresIn:
           type: integer
-        createdAt: 
+        createdAt:
           type: integer
 
     PickingLocation:
       type: object
       properties:
         id:
-          type: string
+          type: integer
+          readOnly: true
+          description: The unique identifier of the picking location
         name:
           type: string
+          description: The name of the picking location.
+          example: "Warehouse 1"
         coords:
           $ref: '#/components/schemas/GeoCoordinates'
         postalCode:
           type: string
+          description: The postal code of the picking location
+          example: "90210"
         addressLine1:
           type: string
+          description: The first line of the address of the picking location
+          example: "123 Main St"
         addressLine2:
           type: string
-        serviceArea:
-          $ref: '#/components/schemas/ServiceArea'
+          description: The second line of the address of the picking location
+          example: "Suite 456"
+        countryCode:
+          type: string
+          format: ISO 3166-1
+          description: The country code of the picking location
+          example: "PL"
+        person_of_contact:
+          type: string
+          description: The person of contact at the picking location
+          example: "John Smith"
+        email:
+          type: string
+          format: email
+          description: The email of the person of contact
+          example: "john.smith@example.com"
+        phone:
+          type: string
+          format: phone
+          description: The phone number of the person of contact
```

Comment:
We should specify if we need a country code here or no. I would say no because we can infer it from the provided Country Code but then it means we need to have that information available and up to date some where.

Reply:
[Discussion result of Engineering + Product]

We should always require the phone indicator because of tourists, expats, etc.

Reply:
[cont]

We should define the phone schema.